src/Mod/Sketcher/Gui/ViewProviderSketch.cpp

```diff
@@ -465,6 +467,9 @@ bool ViewProviderSketch::keyPressed(bool pressed, int key)
             }
             return false;
         }
+    case SoKeyboardEvent::LEFT_SHIFT:

```

Comment:
This is handled when there is a DrawSketchHandler active and when there is not?

One might think that LEFT_SHIFT is too precious to be hardcoded at VP level. I would tend to think that it should be spared for DrawSketchHandler (and its offspring). Maybe I am wrong. Feel free to ping me in the forum.

Reply:
I guess you're right. My new proposal is based on `Mode`. Let's see if you like it. :)